### ISSUE TITLE
Faveo is not showing any incoming emails

### ISSUE THREAD

**Issue by ninoxnino** [ladybirdweb/faveo-helpdesk#347]
The only way that the system will show new emails and customer replies is if I go to email settings and click on "update" button. 

(Note: I don't change any of the settings - I just click the button).

Version: 1.0.9

Is there a way to resolve this?

**Comment by bhanu2217** [comment]
Have you set the cron job?

**Comment by ninoxnino** [comment]
Hi, yes the cron jobs are set on server.

Thanks,
Nino

**Comment by gdaquila** [comment]
Hello 
I think this is a bug. After upgrate to v1.9.1 and removed the original cron job http://SITE/public/readmails and added the new cron job: /usr/bin/php-cli -q  /home/siteName/public_html/artisan schedule:run >> /dev/null 2>&1 the new emails are not readed anymore.

I configured the "cron settings" under admin panel "Settings"

I also tested the above job in a ssh terminal and the execution of the artisan schedule:run command give me the following output (I removed the >>/dev/null ...):
"No schedule commands are ready to run".

To read the mails I needed to restore the original cron with the /public/readmail url.

Regards,
G
